### ISSUE TITLE
Akka routing

### ISSUE THREAD

**Issue by jordi133** [scala-academy/akka-crashcourse#17]
In order for players like the LinearPlayer to be able to participate in many games simultaneously, we would like to let Akka instantiate several instances of this actor by using Routing.

**Comment by TimSoethout** [comment]
Wouldn't it make sense to let the game-actor start the players as child-actors?

**Comment by jordi133** [comment]
Yes that would be possible, the the Game Actor would require two Props-objects to be instantiated. Would then the GameManager be a better candidate for routing?

**Comment by TimSoethout** [comment]
I'm not sure. Routing is useful when having a pool of workers for example, using round-robin or similar to spread some jobs on the routees. 
Maybe if you want to limit the amount of allowed parallel games or something it can make sense to use routers.

**Comment by jordi133** [comment]
Hmm lets take this discussion to gitter, for now I'll adjust this story and move to backlog instead of ready
